doc(1.7.0): add `Replica Auto Balance Disk Pressure Threshold` setting (#924)
longhorn/website: content/docs/1.7.0/references/settings.md

  Longhorn does not forcefully re-schedule the replicas to a zone that does not have enough nodes
  to support even balance. Instead, Longhorn will re-schedule to balance at the node level.

#### Replica Auto Balance Disk Pressure Threshold (%)

> Default: `90`

> Sets the threshold percentage of disk space utilization that triggers replica auto-balance.
> 
> When the threshold percentage is reached, Longhorn automatically rebuilds replicas under disk pressure onto another disk within the same node.
> 
> This setting is effective only under the following conditions:
> - [Replica Auto Balance](#replica-auto-balance) is set to **best-effort**.
> - [Replica Node Level Soft Anti-Affinity](#replica-node-level-soft-anti-affinity) is set to **enabled**.
> - There must be at least one other disk on the node with sufficient available space.

Reviewer: ```suggestion
Percentage of currently used storage that triggers automatic replica rebalancing.

When the threshold is reached, Longhorn automatically rebuilds replicas that are under disk pressure on another disk within the same node.

This setting takes effect only when the following conditions are met:

- [Replica Auto Balance](#replica-auto-balance) is set to **best-effort**.
- [Replica Node Level Soft Anti-Affinity](#replica-node-level-soft-anti-affinity) is set to **enabled**.
- At least one other disk on the node has sufficient available space.
```

  Longhorn does not forcefully re-schedule the replicas to a zone that does not have enough nodes
  to support even balance. Instead, Longhorn will re-schedule to balance at the node level.

#### Replica Auto Balance Disk Pressure Threshold (%)

> Default: `90`

Percentage of currently used storage that triggers automatic replica rebalancing.

When the threshold is reached, Longhorn automatically rebuilds replicas that are under disk pressure on another disk within the same node.

> To disable this feature, set the value to 0.

Reviewer: ```suggestion
To disable this setting, set the value to **0**.
```

This setting takes effect only when the following conditions are met:

> - [Replica Auto Balance](#replica-auto-balance) is set to **best-effort**. To disable this feature when replica auto-balance is set to best-effort, set the value to 0.

Reviewer: ```suggestion
- [Replica Auto Balance](#replica-auto-balance) is set to **best-effort**. To disable this setting (disk pressure threshold) when replica auto-balance is set to best-effort, set the value of this setting to **0**.
```

Reviewer: In this context, "this setting" could mean either, so I specified the setting name to avoid confusing users.

- At least one other disk on the node has sufficient available space.

> This setting is not affected by the [Replica Node Level Soft Anti-Affinity](#replica-node-level-soft-anti-affinity) setting. This means that even if the node soft anti-affinity prevents Longhorn from rebuilding a replica on the same node, this feature will still attempt to rebuild a replica on the same node but on a different disk for migration purpose.

Reviewer: ```suggestion
This setting is not affected by [Replica Node Level Soft Anti-Affinity](#replica-node-level-soft-anti-affinity), which can prevent Longhorn from rebuilding a replica on the same node. Regardless of that setting's value, this setting still allows Longhorn to attempt replica rebuilding on a different disk on the same node for migration purposes.
```
